test/pipeline-runs/go-tests-run.yaml

```diff
@@ -0,0 +1,33 @@
+# Pipelinerun for go unittests
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: go-test-
+  namespace: kf-ci
+  labels:
+    pipeline: go-test
+    repo: gcp-blueprints
```

Comment:
This is a separate repo? Seems it's unnecessary?

Reply:
I fixed it. The labels make it easy to find all pipelines for a given repo/test